src/RoleInterface.php

```diff
@@ -51,7 +51,7 @@ public function addChild($child);
     public function setParent($parent);
 
     /**
-     * @return null|RoleInterface
+     * @return null|RoleInterface|array
```

Comment:
Make sure you document this as a new possibility for implementations starting with the new minor version.

Reply:
I documented the changes in  `doc/book/methods.md`